alphafold3_pytorch/alphafold3.py

```diff
@@ -3377,9 +3378,10 @@ def forward(
         resolved_labels: Int['b n'] | None = None,
         return_loss_breakdown = False,
         return_loss: bool = None,
+        return_present_sampled_atoms: bool = False,
         num_rollout_steps: int = 20,
         rollout_show_tqdm_pbar: bool = False
-    ) -> Float['b m 3'] | Float[''] | Tuple[Float[''], LossBreakdown]:
+    ) -> Float['b m 3'] | Float['l 3'] | Float[''] | Tuple[Float[''], LossBreakdown]:
```

Comment:
did you mean to do a `List[Float['l 3']]` here? (accounting for batched sampling of atom positions)

Reply:
Nope, `l 3` means I sliced out the non-missing atom positions after sampling using `~missing_atom_mask`, which removes the batch dimension and thereby flattens the "present" atom positions into a 2D tensor